lib/mcb/commands/az/apps/pg_dump.rb

```diff
@@ -0,0 +1,20 @@
+name 'pg_dump'
+summary 'backup the psql server for an app'
+option :f, 'target_file', 'target file',
+       argument: :optional
+
+instance_eval(&MCB.remote_connect_options)
+
+run do |opts, _args, _cmd|
+  MCB.load_env_azure_settings(opts)
+  if MCB.requesting_remote_connection? opts
+    MCB::Azure.configure_for_webapp(opts)
+  end
+
+  ENV['PGPASSWORD'] = ENV['DB_PASSWORD']
+  target_file = opts[:target_file]
+  target_file ||= "#{opts[:env]}_#{ENV['DB_DATABASE']}.sql"
```

Comment:
Would it make sense to add the datetime to this?

Reply:
alright alright. done.